src/debug/HyprDebugOverlay.cpp

```diff
@@ -7,8 +7,8 @@ CHyprDebugOverlay::CHyprDebugOverlay() {
     m_pTexture = makeShared<CTexture>();
 }
 
-void CHyprMonitorDebugOverlay::renderData(CMonitor* pMonitor, float µs) {
-    m_dLastRenderTimes.push_back(µs / 1000.f);
+void CHyprMonitorDebugOverlay::renderData(CMonitor* pMonitor, float duration) {
```

Comment:
this is a bannable offense in software development. "duration" without a unit specifier, that is. `durationUs` is ok if you really want to avoid µ

Reply:
heh sorry :), shall remember that. changed it to durationUs.